.circleci/config.yml

```diff
@@ -566,9 +566,6 @@ workflows:
           context: mattermost-mobile-android-pr
           requires:
             - test
-          filters:
-            branches:
-              only: /^(build|android)-pr-.*/
```

Comment:
Please revert this change

Reply:
Should I rename the branch and resend the MR to generate APK for testing?

Reply:
The PR for testing it needs to be done by someone from the org.. ping me when you want me to trigger the build

Reply:
@enahum rebased, could you trigger APK build?

Reply:
@enahum one more ping, sadly the Phabricator Icon asset was missing from the PR :disappointed: